src/protocolsupport/protocol/typeremapper/tile/TileEntityRemapper.java

```diff
@@ -48,6 +48,7 @@
 import protocolsupport.protocol.utils.types.nbt.NBTType;
 import protocolsupport.utils.CollectionsUtils.ArrayMap;
 import protocolsupportbuildprocessor.Preload;
+import static protocolsupport.protocol.typeremapper.tile.TileEntitySkullRemapper.getLegacyData;
```

Comment:
@OnlyBMan I'm not sure about this. We have no other static imports. I haven't seen any formal code rules, but I suggest you don't do static imports.

Reply:
Whoopsie! I didn't notice intellij made it static!